ChrisScianna/ROS-Underwater-RnD
# Improve mission_control quality

# Description

This patch improves and augments `mission_control` test suite, fixing most issues that popped up as result.

## Type of change

Please mark options that are relevant.


# Checklist

# How To Test

Run `mission_control` unit tests sequentially:

```sh
catkin_make -j1 run_tests_mission_control
```

If ran in parallel, some tests flake (see below).
